### PR TITLE
Some new changes

### PR DESCRIPTION
**Pull request by SeaDve**
Added
* area selection mode in wayland
* Minor UI improvements (no major redesign, as requested by latesil)
* Move the settings to the popover
* New sound when you start recording
* replace file selector button
* keyboard shortcuts changes

depends on commits cbfdf73 9b2c60b f6c6db0 5c9d637

### CODE REVIEW

File: data/resources/selectlocation.ui [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Generated with glade 3.38.2 -->
+<interface>
+  <requires lib="gtk+" version="3.24"/>
+  <object class="GtkFileChooserDialog" id="selectlocation">
```
Comment (mathiascode) [inline data/resources/selectlocation.ui:5]: GtkFileChooserNative would allow granting access to folders outside "Videos" in the Flatpak.

Comment (SeaDve) [reply data/resources/selectlocation.ui:5]: I think it is also possible with GtkFileChooserNative,  I tried accessing a folder, as long as it is within home, will work.

Comment (SeaDve) [reply data/resources/selectlocation.ui:5]: I tried and it raise some problems with file saving after restart

**Comment by SeaDve** [comment]
closes #39

**Pull request merged**